### DISCUSSION TITLE
Device library

### DISCUSSION THREAD

**Discussion by robertmain** [netbox-community/netbox#13453] [General]
After inputting all the details for my ER4 (and all it's associated interfaces etc.) I'm now trying to scaffold out a config template for it. It occurs to me that it'd be super neat to have a library of common devices out there you can either copy and paste in or install as a plugin somehow to save other people the same work. 

Curious if there's any interest in this and/or if something like that already exists...

**Comment by jcollie** [comment]
You're in luck, check out the device type library: https://github.com/netbox-community/devicetype-library

**Reply by robertmain** [reply]
Neat, thanks. Be even cooler if they had config templates! I wonder if they accept those as contributions...
